paket.dependencies

```diff
@@ -50,6 +50,7 @@ nuget YoloDev.Expecto.TestSdk
 nuget AltCover
 nuget GitHubActionsTestLogger
 nuget Ionide.LanguageServerProtocol
+nuget Microsoft.Extensions.Caching.Memory
```

Comment:
We can use stale results for completions to get it to be super quick.  However the compiler's TryGetRecentTypeCheck only keeps type checks for the specific sourcetext, which won't really matter for us because we'll be doing incremental type checking on DidChange already